__tests__/store/useChatStore.test.ts

```
@@ -1049,27 +1094,27 @@ describe('useChatStore', () => {
+        .map(() => ({
+          type: 'TYPING_STATUS' as const,
+          tripId,
+          payload: {
+            userId: 'user-2',
+            isTyping: true,
+            username: 'User 2',
+          },
+        }));
 
-      events.forEach(event => useChatStore.getState().handleChatEvent(event));
+      events.forEach((event) => useChatStore.getState().handleChatEvent(event));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)